bin/initial_data.sh

```diff
@@ -1,9 +1,6 @@
-#!/bin/sh
-
-username=$(sed -n '1p' /tmp/user_data)
-password=$(sed -n '2p' /tmp/user_data)
-email=$(sed -n '3p' /tmp/user_data)
```

Comment:
These are now passed from the other script thanks to source

Reply:
It is a pleasure to learn from your code improvements! Thank you!